lockdown/core.py

```diff
@@ -51,6 +56,15 @@ def check(m):
         #        if author_targeted_guild.guild_permissions.manage_channels is False:
         #            return await ctx.send("You don't have `manage_channels` permissions in this guild.")
 
+        lock_check = await self.config.guild(ctx.guild).locked()
```

Comment:
lock check to prevent double run. channel check to verify config settings have channels